### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by afiebig**
Add reference to libserialport and transbankwrap dll in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,9 @@ Primero deberás clonar este repositorio en tu computador:
 ````batch
 git clone https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet-example.git
 ````
+## Dependencias
+
+Asegurate de tener en tu path, o copiar en la carpeta de salida de este proyecto la ultima version de `libserialport.dll` y `TransbankWrap.dll`. Las puedes encontrar incluidas en el ultimo release del SDK github.com/TransbankDevelopers/transbank-pos-sdk-c/releases/latest
```
Comment (gdespirito) [inline README.md]:
```suggestion
Asegúrate de tener en tu path, o copiar en la carpeta de salida de este proyecto la última versión de `libserialport.dll` y `TransbankWrap.dll`. Las puedes encontrar incluidas en el último release del SDK github.com/TransbankDevelopers/transbank-pos-sdk-c/releases/latest
```

File: README.md [hunk 2]
```diff
@@ -35,4 +38,4 @@ El ejemplo viene listo para correr usando Visual Studio:
 4. Clic derecho nuevamente sobre el proyecto `TransbankPosSDKFormExample` y seleccionar la opción Compilar.
 5. Iniciar la aplicación (`ctrl + F5`) `Menú` -> `Debug` -> `Iniciar sin debug`
 
-Si todo ha salido bien deberías ver una aplicación WindowsForm y probar los distintos comandos del POS que tengas conectado a tu equipo.
\ No newline at end of file
+Si todo ha salido bien deberías ver una aplicación WindowsForm y probar los distintos comandos del POS que tengas conectado a tu equipo.
```
Comment (gdespirito) [inline README.md]:
```suggestion
Si todo ha salido bien, deberías ver una aplicación `WindowsForm` y probar los distintos comandos del POS que tengas conectado a tu equipo.
```

**Review by gdespirito — DISMISSED** [review]
👌

File: README.md [hunk 3]
```diff
@@ -24,6 +24,9 @@ Primero deberás clonar este repositorio en tu computador:
 ````batch
 git clone https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet-example.git
 ````
+## Dependencias
+
+Asegúrate de tener en tu path, o copiar en la carpeta de salida de este proyecto la última versión de `libserialport.dll` y `TransbankWrap.dll`. Las puedes encontrar incluidas en el último release del SDK github.com/TransbankDevelopers/transbank-pos-sdk-c/releases/latest
```
Comment (saengate) [inline README.md]:
```suggestion
Asegúrate de tener en tu path, o copiar en la carpeta de salida de este proyecto, la última versión de `libserialport.dll` y `TransbankWrap.dll`. Las puedes encontrar incluidas en el [último release del SDK](https://github.com/TransbankDevelopers/transbank-pos-sdk-c/releases/latest)
```

**Review by saengate — APPROVED** [review]

**Pull request merged**